src/core/public/application/ui/app_container.tsx

```diff
@@ -87,6 +87,8 @@ export const AppContainer: FunctionComponent<Props> = ({
           })) || null;
       } catch (e) {
         // TODO: add error UI
+        // eslint-disable-next-line no-console
+        console.error(e);
```

Comment:
@joshdover Mixed about whether this is acceptable until we actually implements the `TODO` or not. We should probably at least wrap this in `if process.env === 'development'`, wdyt ?

Reply:
We actually just had an SDH where a customer could not see a dashboard because the app failed to mount on a corrupted saved object. Having a stack trace from production would have been a huge help. I'm ok with keep this for now until we get a UI

Reply:
SGTM. We really need to implement client-side logging API at some point to avoid these `console.log`